app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    old_file, new_file = command.split()[1:]
+    to_del = old_file
```

Comment:
u don't need this temp variable (to_del), just use old_file when u wanna remove old file